src/main/scala/org/broadinstitute/dsde/vault/common/directives/VersioningDirectives.scala

```diff
@@ -1,17 +1,26 @@
 package org.broadinstitute.dsde.vault.common.directives
 
-import shapeless.{HList, ::, HNil}
-import spray.routing.{PathMatcher, Directive}
+import shapeless.{::, HList, HNil}
 import spray.routing.directives.BasicDirectives._
 import spray.routing.directives.PathDirectives._
+import spray.routing.{Directive, PathMatcher}
 
 object VersioningDirectives {
-  val optionalVersion: Directive[Option[Int] :: HNil] =
-    pathPrefix("v" ~ IntNumber).map(Option(_)) | provide(None)
+  val optionalVersion: Directive[Option[Int] :: HNil] = pathPrefix("v" ~ IntNumber).map(Option(_)) | provide(None)
```

Comment:
Please document and provide usage examples for each of these.

Are any intended for internal use only?  Mark as private/protected as appropriate.